packages/gatsby-plugin-fastify/src/plugins/decorators.ts

```diff
@@ -3,15 +3,19 @@ import fp from "fastify-plugin";
 import { moduleHeaderDecorator, setHeaderDecorator } from "../utils/headers";
 
 import type { FastifyPluginAsync } from "fastify";
+import type { ISSRData } from "gatsby/dist/utils/page-ssr-module/entry";
 
 declare module "fastify" {
   interface FastifyReply {
     appendModuleHeader: typeof moduleHeaderDecorator;
     setHeader: FastifyReply["header"];
+    pageQueryData: ISSRData | null;
   }
 }
 
 export const implementUtilDecorators: FastifyPluginAsync = fp(async (fastify) => {
   fastify.decorateReply("setHeader", setHeaderDecorator);
   fastify.decorateReply("appendModuleHeader", moduleHeaderDecorator, ["setHeader"]);
+  // eslint-disable-next-line unicorn/no-null
```

Comment:
I set it to `null` as suggested in [fastify docs](https://www.fastify.io/docs/latest/Reference/Decorators/): 

> Note that it is important to keep the initial shape of a decorated field as close as possible to the value intended to be set dynamically in the future. Initialize a decorator as a `''` if the intended value is a string, and as `null` if it will be an object or a function.

I'm not sure the intention/benefits of the linting rule, but I assumed to go with the vendor suggestion. Which way do you want to land?

Reply:
I changed this around a bit in 9fcd9b1dcd51196968efb43ab4d923599a98cd20.

Since the type is now `ISSRData["serverDataHeaders"]` which includes `| undefined` I tried switching the initialization to use `undefined` but the `lint-staged` ended up rewriting it to `fastify.decorateReply("serverDataHeaders");` which then causes a `too few arguments` issue so I amended it back to using `null`. I think we need to keep it `null` but let me know if you think otherwise.

Reply:
Yeah, tell should be fine. Rules are meant to be broken. https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-null.md has some info on the reasoning if you're curious.